### ISSUE TITLE
Example code in README.md imports non-existent version

### ISSUE THREAD

**Issue by JamesxX** [tingerrr/hydra#1]
`#import "@preview/hydra:0.0.1": hydra
` instead of the correct `#import "@preview/hydra:0.1.0": hydra`

**Comment by tingerrr** [comment]
Indeed, that was already fixed on the PR, but I forgot it here. Thanks!
